gno.land/pkg/gnoland/app.go

```diff
@@ -20,34 +22,66 @@ import (
 	"github.com/gnolang/gno/tm2/pkg/store/iavl"
 )
 
+type CustomAppConfig struct {
+	DB                    dbm.DB
+	GnoRootDir            string
```

Comment:
We have multiple "root directories" in the repository. Please add a comment specifying the specific root directory or consider using a more distinctive name.